Open-Source-Pundits/nutshell
# Mb render news page

# Description

Some styling. Added delete functionality to news page, and form shows up when create form button is clicked.

Fixes #23 #24

## Type of change


# Testing Instructions

- `git fetch --all`
- `git checkout mb-render-news-page`
- copy database.example file data to your db to make sure delete functionality works
- make sure create new form link brings up a form.

# Checklist:
